web/src/main/webapp/xslt/ui-search/portal-list.xsl

```diff
@@ -64,10 +64,10 @@
       </div>
     </div>
     <div class="container gn-info-list-blocks">
-      <ul class="row gn-info-list">
+      <ul class="row list-group gn-info-list">
         <xsl:for-each select=".//sources/record[type = 'subportal']">
           <xsl:sort select="label/*[name() = $lang]"/>
-          <li>
+          <li class="list-group-item panel panel-default gn-card">
```

Comment:
![image](https://user-images.githubusercontent.com/1701393/111458268-89288b80-8719-11eb-9409-868aa0fca49a.png)

for me, the layout was better before for the portal list. Now the description is not displayed. and the border is quite "heavy" . But that's only my preference.

Reply:
I'll see what I can do

Reply:
> I'll see what I can do

You can leave it like this. It is just my point of view (and some of GeoNetwork users like https://galliwasp.eea.europa.eu/catalogue/srv/api/sources)

Reply:
It has my attention, showing the description is a good addition